Assets/GothicVR/Scripts/Npc/RoutePlanner.cs

```diff
@@ -0,0 +1,16 @@
+using PxCs.Data.WayNet;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace GVR
+{
+    public class RoutePlanner : MonoBehaviour
+    {
+        public List<PxWayPointData> CreatePath(string start, string end)
+        {
+            return null;
```

Comment:
Why this method, if it won't return anything?

If you wanted to start the route planner here, I suggest you use 
Assets/GothicVR/Scripts/Manager/WayNetHelper.cs
as it is already intended to be used for WayNet related topics at runtime.

Reply:
@Frozenenenen what was your original idea for this?

Reply:
Updated.